### PR TITLE
Sandhu/aug archive n mongodb

### PR DESCRIPTION
**Pull request by sandhujasmine**
(WIP) updating the AnacondaRepository runbook for the Aug archive. Only updating the *.rst files at present so updated binary files are not checked back in yet - this is more to make it easier for code review and it is easy to see if there are any merge conflicts here.

### CODE REVIEW

**Comment by bkreider** [comment]
Can you add the size of the tarballs in the table listing the components?  I had to manually check them and then run the script after my digital ocean instance ran out of disk space.

File: AnacondaRepository.rst [hunk 1]
```diff
@@ -102,48 +114,192 @@ the steps below have two sections: **Air Gap Installation** and
...
+      'linux-64-R-Packages-2016-08-03.list'
+      'linux-64-R-Packages-2016-08-03.md5'
+      'linux-64-R-Packages-2016-08-03.tar'
+      'miniconda-64bit-2016-08-03.tar')
+
+  for file_ in "${DL[@]}"
+  do
+    CMD="curl -O -C - $PREFIX$file_"
+    echo $CMD > cout.txt
+    $CMD >> cout.txt
```
Comment (bkreider) [inline AnacondaRepository.rst]: That should be ">" cout.txt.  The double ">>" wipes the file each time; the single '>' will append.

Comment (sandhujasmine) [inline AnacondaRepository.rst]: @bkreider - I think you have it backwards
```
$ echo "test" > test.txt
$ echo "add to test" >> test.txt
$ cat test.txt 
test
add to test
```

The [`>>`](https://bash.cyberciti.biz/guide/Appending_redirected_output) will append while [`>`](https://bash.cyberciti.biz/guide/Redirection_of_standard_output) will clobber.

**Comment by sandhujasmine** [comment]
> Can you add the size of the tarballs in the table listing the components? 

Added a column for this

File: AnacondaRepository.rst [hunk 2]
```diff
@@ -102,113 +120,266 @@ the steps below have two sections: **Air Gap Installation** and
...
+After downloading, expand the tarballs such that they they are consistent with full archive and the instructions from here on down.
+Running the commands from the script below will do this - set the `$INSTALLER_PATH` correctly for your system. This will take sometime to expand the archives.
+
+.. code-block:: bash
+
+  #!/bin/bash
+  #
+  # get_linux_installers.sh - to obtain all components for Linux packages and AE-Repo installation
+  # expect INSTALLER_PATH env var is set before script is run
+
```
Comment (bkreider) [inline AnacondaRepository.rst]: Can you add something like this so it doesn't try and run with bad input?

```
if [ -z "$INSTALLER_PATH"]
then
    echo "INSTALLER_PATH not set or empty.  Please set."
    exit 1
fi
mkdir -p $INSTALLER_PATH
```

It should also say how to set that variable using `export INSTALLER_PATH=/installer`.

File: AnacondaRepository.rst [hunk 3]
```diff
@@ -227,12 +398,16 @@ Give the anaconda-server user ownership of directories
...
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Switch to the Anaconda Repository administrator account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Switch account, and set `$INSTALLER_PATH` environment variable.
 
 ::
 
     sudo su - anaconda-server
+    INSTALLER_PATH=<set to path of air gap data>
```
Comment (bkreider) [inline AnacondaRepository.rst]: Can you change this to `export INSTALLER_PATH`?

**Pull request merged**